msbuild/Xamarin.Localization.MSBuild/TranslatedAssemblies/MSBStrings.cs.resx

```diff
@@ -1,1180 +1,1205 @@
-﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
This PR is changing EOL again.

Reply:
It also looks like from inspecting the file that it still has the BOM included as this PR should have removed it (https://github.com/xamarin/xamarin-macios/pull/18042)
![image](https://user-images.githubusercontent.com/50846373/233142135-e57968aa-1121-415d-a193-d3505edab94d.png)

Reply:
Also, here is an example of one of the files using CRLF endings (from my mac) 
![image](https://user-images.githubusercontent.com/50846373/233142453-2ad58d08-a00e-4ba2-865d-93989091500c.png)

Reply:
Followed up with Loc team here: https://ceapex.visualstudio.com/CEINTL/_workitems/edit/816066#12372395